src/containers/footer-lists/footer-lists.js

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+
+import { Link } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import { useStyles } from './footer-lists.styles';
+
+import {
+  URL_LANGUAGE,
+  FOOTER_INFORMATION,
+  FOOTER_CONTACTS,
+  FOOTER_CATALOGS
+} from '../../configs';
+import { getCategories } from '../../redux/categories/categories.actions';
+
+const FooterLists = ({ getCategories, list, language = 0 }) => {
+  const styles = useStyles();
+
+  useEffect(() => {
+    getCategories();
+  }, [getCategories]);
+
+  const getCategoryURL = (category) => {
+    const [filteredCategory] = category.filter(
+      (item) => item.lang === URL_LANGUAGE
```

Comment:
Can `URL_LANGUAGE` be changed?

Reply:
Here is resolved this
https://github.com/horondi/horondi_client_fe/pull/53